models/image.py

```diff
@@ -9,13 +9,15 @@
 
 import sqlalchemy as sa
 from sqlalchemy import orm
+import psycopg2.extras
 
 from sqlalchemy.dialects.postgresql import JSONB
 from sqlalchemy.dialects.postgresql import UUID as sqlUUID
 from sqlalchemy.ext.declarative import declared_attr
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.schema import CheckConstraint
+from sqlalchemy.sql import or_, and_
```

Comment:
So... I've resisted linters because the net-favorite one (black) completely destroys code readability in pursuit of rigid consistency.  But, there are some that you can configure to not destroy your code, but still catch things like this.  I will look into adding using one of those into the PR checks, so we will always get a report on that.